keylime/tpm/tpm_main.py

```diff
@@ -303,6 +305,9 @@ def __create_ek(self, asym_alg=None):
                     "-P",
                     owner_pw,
                 ]
+            else:
+                logger.error("Internal error")
+                sys.exit()
```

Comment:
Conflict with #1176

Reply:
This is a leftover, I will fix it. Thanks!